polkadot/node/subsystem-util/src/inclusion_emulator/mod.rs

```diff
@@ -521,18 +520,13 @@ impl ConstraintModifications {
 /// The prospective candidate.
 ///
 /// This comprises the key information that represent a candidate
-/// without pinning it to a particular session. For example, everything
-/// to do with the collator's signature and commitments are represented
-/// here. But the erasure-root is not. This means that prospective candidates
+/// without pinning it to a particular session. For example commitments are
+/// represented here. But the erasure-root is not. This means that prospective candidates
 /// are not correlated to any session in particular.
 #[derive(Debug, Clone, PartialEq)]
```

Comment:
Nit: I see this derives `PartialEq` is there any risk to have two different candidates being equal ?

Reply:
They should have different commitments, if not it is truly the same candidate.